src/Button/style.css

```diff
@@ -0,0 +1,54 @@
+button {
+    border: none;
+    border-radius: 4px;
+    text-transform: uppercase;
+    cursor: pointer;
+    height: 57px;
+    width: 180px;
+}
+
+.confirm {
+    color: var(--color-5);
+    background-color: var(--color-1);
+}
+
+.cancel {
+    color: var(--color-1);
+    border: 1px solid var(--color-1);
+    background-color: transparent;
+    border-radius: 4px;
+}
+
+.close {
+    background-color: transparent;
+    color: var(--color-5);
+    width: 21px;
+    height: 21px;
+    padding: 0;
+    font-size: 21px;
+}
+
+.menu {
+    background-color: var(--color-3);
+    color: white;
```

Comment:
Let's move all colors to variables.

Reply:
Added more color variables to styles in the latest commit